.github/workflows/lint.yaml

```diff
@@ -1,34 +0,0 @@
-name: Lint
```

Comment:
why does this PR delete this?

Reply:
I used gitpod ide. Must have been the environment that created that.

Reply:
can you remove it?